src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -5,12 +5,31 @@ import { withDesign } from "storybook-addon-designs";
 
 import Checkbox from "./Checkbox";
 
+const argsBooleanType = (defaultValue = false) => ({
+  control: { type: "boolean" },
+  table: { defaultValue: { summary: defaultValue } },
+});
```

Comment:
Should this be made global?

Reply:
Yes. In a later PR I did make it global but did not for this PR. So I can deal with the merge conflicts later on.